src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="box-container">
+        <div className="result-screen">0</div>
```

Comment:
- [ ] Kindly ensure your design matches the given template.

  Your Implementation | Design template
  :---:|:---:
  ![ozb-1](https://user-images.githubusercontent.com/36542986/164003043-0121d82b-7ffc-4c15-87a1-6968f43231bd.jpg) | ![ozb-2](https://user-images.githubusercontent.com/36542986/164003050-3a259ba2-7eb6-468c-8bf7-5dcf25bddf36.jpg)

Reply:
updated